backend/app/core/weather_client.py

```
@@ -0,0 +1,252 @@
+            return list(forecasts.values())
+    except Exception:
+        return None
+
+
+def _generate_mock_weather(sensor_data: dict | None = None) -> dict:
+    """센서 데이터 기반 mock 기상 데이터 생성."""
+    now = datetime.now(KST)
+    kst_hour = now.hour
+
+    if sensor_data:
+        base_temp = sensor_data.get("temperature", 22) - random.uniform(1, 4)
```

Ruff (0.15.9)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)